### ISSUE TITLE
Scope GravityView_Delete_Entry->process_delete()

### ISSUE THREAD

**Issue by dcavins** [gravityview/GravityView#624]
Hi all-

GravityView is causing conflicts with BuddyPress because it is catching all URLs with `?action=delete` in  `GravityView_Delete_Entry->process_delete()` at the `wp` action.

Changing the conditional to
```php
if ( $_GET('action') === 'delete' && isset( $_GET['entry_id'] ) )
``` 
would avoid my specific problem, but still relies on pretty generic query argument names. Since you know when this action needs to run (and I'm not even sure if this is a front-end or admin-only thing), I'm sure you've got a good idea of how to limit it with a conditional or, if it could run anywhere, making the action string more specific.

Here's an example of a url that `process_delete()` attempts to catch (and then drops to WP's bad nonce "are you sure you want to do that" screen):
http://mywebsite.org/members/member-name/notifications/unread/?action=delete&notification_id=18740&_wpnonce=123456789ab

Thanks!

**Comment by zackkatz** [comment]
@dcavins Good catch, thanks! Please test the patch and let me know if it works for you.

**Comment by dcavins** [comment]
Hi Zack- 

Yep, that works in my case.

Thanks for your quick response!

-David

**Comment by zackkatz** [comment]
@dcavins - Just released 1.16.2, which includes this update. Thanks again for reporting.
